### PR TITLE
Rewrite issue template

### PR DESCRIPTION
**Pull request by Nutomic**
The old order of items didn't really make sense to me.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -2,15 +2,23 @@
...
+3.
+
+
+**Additional information**
+* PeerTube version or URL:
+* Browser name/version:
+* NodeJS version:
+
+* Link to browser console log if useful:
+* Link to server log if useful (journalctl or /var/www/peertube/storage/logs):
```
Comment (Jorropo) [inline .github/ISSUE_TEMPLATE.md]: Maybe if /var/www/peertube/storage/logs finish with a / that will be more clear (for indicate that the logs are in the folder, not a single file).

Comment (Chocobozzz) [reply .github/ISSUE_TEMPLATE.md]: :+1:

Comment (Nutomic) [reply .github/ISSUE_TEMPLATE.md]: Makes sense, fixed :)

**Comment by Chocobozzz** [comment]
> I would even add another section below the "bug report form", for "features requests" because they happen and should be a bit more structured too, if possible

I would prefer to keep the template as simple as possible, and we need to focus on bugs (that are the most difficult to reproduce and to fix).

**Pull request merged**